app-compat-travis.yml

```diff
@@ -6,19 +6,26 @@ services:
 
 matrix:
     include:
-        - env:
+        - os: linux
+          env:
             - PROJECT_NAME=kshdb
               GIT_URL="https://github.com/rocky/kshdb.git"
               COMMANDS="git checkout att-ksh93v-testing; ksh ./autogen.sh; make test"
 
+        - os: linux
+          env:
+            - PROJECT_NAME=yash
+              GIT_URL="https://github.com/siteshwar/yash.git"
```

Comment:
Should this have a "TODO" comment explaining why we're using your personal clone and that we hope to use the unmodified yash tests?

Reply:
I have added a list of tests to run and testrunner through https://github.com/siteshwar/yash/commit/fbe0139c2dbe0242fce4f00978cefd35caba42a8. I can leave a note here to clarify why I am using my clone.

Reply:
and opened #859 to investigate failing tests.